.github/workflows/code-formatting.yml

```diff
@@ -71,13 +73,17 @@ jobs:
             commit: --signoff
 
       - name: pylint
-        if: ${{ !contains( github.event.pull_request.labels.*.name, 'skip-docs') }}
+        if: ${{ steps.changed-files.outputs.any_changed == 'true' && !contains( github.event.pull_request.labels.*.name, 'skip-docs') }}
+        env:
+          # only *.py files included
+          CHANGED_FILES: "${{ steps.changed-files.outputs.all_changed_files }}"
         run: |
           pip install pylint
-          
-          CHANGED_FILES=$(git diff --name-only --diff-filter=d --merge-base origin/main nemo/ | grep '\.py$' || true)
 
           ADDITIONAL_PYLINT_ARGS=()
           echo ${{ github.event.pull_request.labels.*.name }}
 
-          pylint ${ADDITIONAL_PYLINT_ARGS[@]} $CHANGED_FILES
\ No newline at end of file
+          pylint ${ADDITIONAL_PYLINT_ARGS[@]} $CHANGED_FILES || \
+            { echo "Pylint failed. In case of long strings, format docstrings manually. "; \
+              echo "To ignore docstring formatting errors, add \"skip-docs\" label"; \
```

Comment:
I’m worried that with this hint, setting the label will become the new standard. Can we leave this for now?

Reply:
That would be awesome

Reply:
done